plugins/node/opentelemetry-instrumentation-mongodb/src/types.ts

```diff
@@ -161,3 +161,15 @@ export enum MongodbCommandType {
   COUNT = 'count',
   UNKNOWN = 'unknown',
 }
+
+// https://github.com/mongodb/node-mongodb-native/blob/v4.2.2/src/cmap/connection.ts
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
nit: do we still need to disable eslint here?

Reply:
Right! fixed :)